include/llbuild/BuildSystem/SwiftTools.h

```diff
@@ -17,14 +17,26 @@
 
 #include "llvm/ADT/StringRef.h"
 
+#include "llbuild/BuildSystem/BuildDescription.h"
+
 #include <memory>
 
 namespace llbuild {
 namespace buildsystem {
 
-class Tool;
-
-std::unique_ptr<Tool> createSwiftCompilerTool(StringRef name);
```

Comment:
If we are moving this to be a builtin tool, I think we can drop this header completely and just define this tool in BuildSystem.cpp, as we do the other builtin ones.